execution-engine/blessed-contracts/pos/src/queue.rs

```diff
@@ -56,17 +60,47 @@ impl ToBytes for QueueEntry {
     }
 }
 
-pub struct Queue(pub Vec<QueueEntry>);
+pub trait QueueProvider {
+    /// Reads bonding queue.
+    fn read_bonding() -> Queue;
 
-impl Queue {
-    pub fn read_bonding() -> Queue {
+    /// Reads unbonding queue.
+    fn read_unbonding() -> Queue;
+
+    /// Writes nbonding queue.
```

Comment:
Typo: "`unbonding`"

Reply:
It's `bonding` actually but it's a typo nevertheless. Thanks.